src/findFiles.ts

```diff
@@ -18,9 +18,16 @@ async function findFileReadDir(
   const directoryFiles = await fs.readdir(cwd);
 
   // Filter the diretory files to only thoose with passed extenison and `.js` or `.jsx`
-  const matchedFiles = directoryFiles.filter((directoryFileName) =>
-    [...extensions, ...JS_EXTS].includes(directoryFileName.split(fileName)[1]),
-  );
+  const matchedFiles = directoryFiles.filter((directoryFileName) => {
+    for (let extension of [...extensions, ...JS_EXTS]) {
```

Comment:
Any reason that this isn't `for(const extension of [...extensions, ...JS_EXTS]) {`